Infinispan featured in Red Hat Developers (#215)
infinispan/infinispan.github.io: _posts/2021-05-31-infinispan-redhat-summit-quarkus.adoc

---
layout: blog
title: Infinispan (Red Hat Data Grid) featured in Red Hat Developers
permalink: /blog/:year/:month/:day/infinispan-redhat-summit-quarkus
date: '2021-05-31T12:00:00.000-00:00'
author: karesti
tags: [ "redhat", "datagrid", "quarkus", "cross-site-replication"]
---

Dear Infinispan Community,

> The Infinispan team is pleased to share the article published the 28th May 2021 in the Red Hat Developers blog.

Reviewer: ```suggestion
The Infinispan team is pleased to share an article published over on the Red Hat Developer blog.
```

> First part of two articles, this blog post focuses on how Red Hat Data Grid, which is built on Infinispan,
> was used to build the leaderboard of the Battleship game this year.

Reviewer: ```suggestion
Part one of a two-part series of articles, this blog post focuses on how Data Grid, which is built on Infinispan,
was used to create a leaderboard for an online Battleship game that featured at this year's Red Hat Summit Keynote.
```

*You can read the blog post https://developers.redhat.com/articles/2021/05/28/building-real-time-leaderboard-red-hat-data-grid-and-quarkus-hybrid-kubernetes[here]*

> Technologies involved:

Reviewer: ```suggestion
Technologies featured in our keynote demo:
```

* Quarkus and Infinispan Client Extension
* Additional Quarkus extensions: RestEasy, Websockets, Scheduler
* Infinispan Query

> * Infinispan Cross-Site replication

Reviewer: ```suggestion
* Infinispan Cross-Site Replication
```

* Infinispan Kubernetes Operator

> Enjoy the reading!

Reviewer: ```suggestion
Enjoy your reading!
```
